modules/ob-uninstalling-the-builds-operator.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// builds/uninstalling-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id='ob-uninstalling-the-builds-operator_{context}']
+= Uninstalling the {builds-title} Operator
+
+You can uninstall the {builds-title} Operator by using the *Administrator* perspective in the web console.
+
+
+.Procedure
+
+. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for the *{builds-title}* Operator.
```

Comment:
As per ISG, start the step with the purpose of action so that users choose whether to read ahead or not.

```suggestion
. Navigate to *Operators* -> *OperatorHub*.
. In the *OperatorHub* page, search for the *{builds-title}* Operator by using the *Filter by keyword* box .
```